### PR TITLE
Adds link to https://katacontainers.io/community/ in Community section.

### PR DESCRIPTION
**Pull request by mvincerx**
Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

**Comment by grahamwhaley** [comment]
Hi @mvincerx The CI is complaining, as you'll need to meet the requirements as set out in the CONTRIBUTING guide listed below:

```
Found 1 commit between commit f49b1cc19cfac676d15a00295e94531f970c0a42 and branch master
ERROR: Commit f49b1cc19cfac676d15a00295e94531f970c0a42: Failed to find subsystem in subject: "Adds link to https://katacontainers.io/community/ in Comunity section."
ERROR: checkcommits failed. See the document below for help on formatting
commits for the project.
https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format
```

Personally, I'd try to make that addition a bit more directive - it's a bit 'implicit' right now, feeling like an incomplete sentence that is completed by it being a link. It reads odd to me. How about:

> Kata Containers is working to build a global, diverse and collaborative community. Anyone who is interested in supporting the technology is welcome to participate. We are seeking different expertise and skills, ranging from development, operations, documentation, marketing, community organization and product management. Learn how to contribute on the [Community pages](https://katacontainers.io/community/).

**Comment by mvincerx** [comment]
Would you mind adding @CraigSterrett as another reviewer?

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Kata Containers combines technology from [Intel® Clear Containers](https://gith
 
 # Community
 
-Kata Containers is working to build a global, diverse and collaborative community. Anyone who is interested in supporting the technology is welcome to participate. We are seeking different expertise and skills, ranging from development, operations, documentation, marketing, community organization and product management.
+Kata Containers is working to build a global, diverse and collaborative community. Anyone who is interested in supporting the technology is welcome to participate. Learn how to contribute on the [Community pages](https://katacontainers.io/community/). We are seeking different expertise and skills, ranging from development, operations, documentation, marketing, community organization and product management.
```
Comment (jodh-intel) [inline README.md]: Thanks for raising this @mvincerx. I'm not sure if this link is the correct one though. Or, maybe https://katacontainers.io/community/ needs to be reworked: One that page under "How to Contribute", the first bullet is "
"Installation Guides", whereas I don't see a link to our main contributing guide:

- https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md

/cc @jimmytipit.

**Comment by mvincerx** [comment]
GH-145 was raised because @CraigSterrett wanted a clear path for the user to find the Architecture Committee meetings.  I included this link https://katacontainers.io/community/ because it allows you to find that Committee meeting info. It's not the first header, but it's further down the page.

File: README.md (hunk 1 above)
Comment (jodh-intel) [reply README.md:29]: I think we should land this now - we can discuss URL tweaks on follow-up PRs...

**Pull request merged**